src/connection_pool.rs

```diff
@@ -5,32 +5,32 @@
 pub mod types;
 
 use std::cmp::Ordering;
-use std::collections::HashMap;
+use std::collections::{HashMap};
 use std::marker::PhantomData;
 use std::ops::{Deref, DerefMut};
 use std::sync::atomic::AtomicBool;
 use std::sync::atomic::Ordering as AtomicOrdering;
 use std::sync::mpsc::{channel, Receiver, Sender};
 use std::sync::{Arc, Barrier};
 use std::{thread, time};
+use std::time::Duration;
 
 use slog::{debug, error, info, warn, Logger};
 
 use crate::backend::{Backend, BackendKey};
 use crate::connection::Connection;
 use crate::connection_pool::types::{
     ConnectionCount, ConnectionData, ConnectionKeyPair, ConnectionPoolOptions,
-    ConnectionPoolState, ConnectionPoolStats, ProtectedData, RebalanceCheck
+    ConnectionPoolState, ConnectionPoolStats, ProtectedData, RebalanceCheck, ShuffleCollection,
 };
 use crate::error::Error;
 use crate::resolver::{
     BackendAction, BackendAddedMsg, BackendMsg, BackendRemovedMsg, Resolver
 };
 
-// General TODOs
-// * Decoherence
-
 const DEFAULT_REBALANCE_ACTION_DELAY: u64 = 100;
+const DEFAULT_DECOHERENCE_INTERVAL: u64 = 60;
```

Comment:
It threw me off observing the behavior that the units of delay is milliseconds and the units for interval is seconds. Could you comment to that effect here and above each field in the `ConnectionPool` struct?